pages/services/cassandra/2.6.0-3.11.4/getting-started/index.md

```diff
@@ -0,0 +1,187 @@
+
+## Launch two {{ model.techShortName }} clusters
+
+Launch the first cluster with the default configuration:
+
+```shell
+dcos package install {{ model.serviceName }}
+```
+
+Create an `options.json` file for the second cluster that specifies a different service name and data center name:
+
+```json
+{
+  "service": {
+    "name": "{{ model.serviceName }}2",
+    "data_center": "dc2"
+  }
+}
+```
+
+Launch the second cluster with these custom options:
+```
+dcos package install {{ model.serviceName }} --options=<options>.json
+```
+
+## Get the seed node IP addresses
+
+<p class="message--note"><strong>NOTE: </strong>If your {{ model.techShortName }} clusters are not on the same network, you must set up a proxying layer to route traffic.</p>
+
+Get the list of seed node addresses for the first cluster:
+
+```shell
+dcos {{ model.serviceName }} endpoints node
+```
+
+Alternatively, you can get this information from the scheduler HTTP API:
+
+```json
+DCOS_AUTH_TOKEN=$(dcos config show core.dcos_acs_token)
+DCOS_URL=$(dcos config show core.dcos_url)
+curl -H "authorization:token=$DCOS_AUTH_TOKEN" $DCOS_URL/service/{{ model.serviceName }}/v1/endpoints/node
+```
+
+Your output will resemble:
+
+```
+{
+  "address": [
+    "10.0.1.236:9042",
+    "10.0.0.119:9042"
+  ],
+  "dns": [
+    "node-0-server.{{ model.serviceName }}.autoip.dcos.thisdcos.directory:9042",
+    "node-1-server.{{ model.serviceName }}.autoip.dcos.thisdcos.directory:9042"
+  ],
+  "vip": "node.{{ model.serviceName }}.l4lb.thisdcos.directory:9042"
+}
+```
+
+Note the IP addresses in the `address` field.
+
+Run the same command for your second {{ model.techShortName }} cluster and note the IP addresses in the `address` field:
+
+```
+dcos {{ model.serviceName }} endpoints node --name={{ model.serviceName }}2
+```
+
+## Update configuration for both clusters
+
+Create an `options.json` file with the IP addresses of the first cluster (`{{ model.serviceName }}`):
+
+```json
+{
+  "service": {
+    "remote_seeds": "10.0.1.236,10.0.0.119"
+  }
+}
+{
+  {}
+}
+```
+
+Update the configuration of the second cluster:
+
+```
+dcos {{ model.serviceName }} update start --options=options.json --name={{ model.serviceName }}2
+```
+
+Perform the same operation on the first cluster, adding the IP addresses of the second cluster's seed nodes to the `service.remote_seeds` field. Then, update the first cluster's configuration: `dcos {{ model.serviceName }} update start --options=options.json`.
+
+Both schedulers will restart after the configuration update, and each cluster will communicate with the seed nodes from the other cluster to establish a multi-data-center topology. Repeat this process for each new cluster you add.
```

Comment:
```suggestion
Both schedulers will restart after the configuration update, and each cluster will communicate with the seed nodes from the other cluster to establish a multi-data center topology. Repeat this process for each new cluster you add.
```

Reply:
Mention the `list of steps` you want an user to repeat for this process.

Reply:
list of steps are already mentioned in above 3 lines, if customer is reading this 3 lines he can easily understand what its meant by "this" .